lib/RuntimeLib.lua

```diff
@@ -974,4 +974,32 @@ function TS.opcall(func, ...)
 	end
 end
 
+TS.TRY_RETURN = 1
```

Comment:
I think numbers might be dangerous here? We should instead use tables to create unique references.
```Lua
TS.TRY_RETURN = {}
TS.TRY_BREAK = {}
TS.TRY_CONTINUE = {}
```

Reply:
The generated code will only ever return TRY_RETURN, TRY_BREAK, TRY_CONTINUE, or nil. My primary reason for using these constants at all is to make the code readable to developers using this from the Lua side.

If we want unique values here, it means we're either worried about human error from manually using `TS.try`, or we're worried about logic errors in the compiler letting other returns slip through.

I think it's reasonable to be concerned about these things and I think using tables for unique values here is reasonable. If these things are a concern, then here's another I'd like your opinion on:

Right now `transformTryFunction` generates code that look like (with irrelevant parts omitted, variables renamed, and comments added):
```lua
local exitType, returns = TS.try(....)
if exitType == TS.TRY_RETURN then
    return unpack(returns)
elseif exitType == TS.TRY_BREAK then
    break
elseif exitType then -- comparison omitted since this *has to* be continue according to the typescript code this was compiled from
    continue
end
```
This equality check omission on the final elseif statement is done in all cases. If we're worried about values other than these three constants slipping through and being returned then we should be doing a full comparison on that final elseif statement, right?

Reply:
This sounds reasonable to me 👍